pkg/storaged/stratis/pool.jsx

```diff
@@ -310,15 +493,23 @@ export function make_stratis_pool_page(parent, pool) {
             },
             {
                 title: _("Delete pool"),
-                action: () => delete_pool(pool, pool_card),
+                action: () => delete_pool(pool, card),
                 danger: true,
             },
         ],
     });
 
-    const fsys_card = new_card({
+    if (pool.Encrypted)
```

Comment:
I'd keep the separate declarations of the cards. Otherwise it becomes confusing to debug and also can be confusing to create new cards with `next` pointing to cards that don't exist anymore technically.

Reply:
Ok, done.